README.md

```diff
@@ -1,42 +1,157 @@
-# Astronaut Health Monitoring Application
+# SecureSpace: Astronaut Health Monitoring Application
 
 **A group project for the post-graduate Computer Science programs at the University of Essex Online.**
 
 Module: _Secure Software Development_\
-Team Name: _SecureSpace_\
 Members: _Bradley Graham, Michael Sammueller, Tomas Mestanza & Rachel Doherty_
 
-## Description
+### Table of Contents
```

Comment:
we should probably stick to the markdown convention, which is to use two hashes for a title after a one hash title if that makes sense